docs/zh/navigation.yml

```
@@ -166,6 +166,7 @@ nav:
                   - 如何选择运行时: kpanda/user-guide/clusters/runtime.md
                   - 集群版本支持范围: kpanda/user-guide/clusters/cluster-version.md
                   - 接入 rancher 集群: kpanda/user-guide/clusters/integrate-rancher-cluster.md
+                  - 如何在集群中部署第二调度器 scheduler-plugins: kpanda/user-guide/clusters/cluster-scheduler-plugin.md
```

```suggestion
                  - 在集群中部署第二调度器: kpanda/user-guide/clusters/cluster-scheduler-plugin.md
```
nav 这边能短一些吗，build 后占 2-3 行不太好看

---

可以的，文章标题不变就行